facturacion_mexico/cfdi_recibidos/services/concept_classifier.py

```
@@ -0,0 +1,98 @@
+
+	doc.db_set("status", "Falta clasif.")
+	return _result(
+		"falta_clasif",
+		total,
+		matched,
+		unmatched,
+		f"Sin regla para: {', '.join(set(unmatched_keys))}",
+	)
+
+
+def _find_rule(company: str, supplier_rfc: str, sat_product_key: str) -> dict | None:
```

Ruff (0.15.13)

[warning] Unused function argument: `company`

(ARG001)